test/quic_multistream_test.c

```diff
@@ -3150,6 +3328,10 @@ static const struct script_op *const scripts[] = {
     script_41,
     script_42,
     script_43,
+    script_44,
+    script_45,
+    script_46,
+    script_47
```

Comment:
Worth noting that my assorted tests PR goes up to script 67. So either this merges first and I have to update that and we wait 24h again or that one merges first and this one has to be changed again.

If you wanted you could renumber these now to start at 68. Git will probably not merge them automatically because the lines will be next to each other but the person merging could reasonably resolve that in a trivial manner.

Renumbering these after things merge has been a pain, I'm probably going to allow tests to be named in the future.